import "game_commands.proto";
message Command_Shuffle {
    extend GameCommand {
        optional Command_Shuffle ext = 1003;
    }
}

